pkg/cloudprovider/providers/azure/azure_backoff.go

```diff
@@ -128,14 +128,14 @@ func (az *Cloud) CreateOrUpdateInterfaceWithRetry(nic network.Interface) error {
 }
 
 // DeletePublicIPWithRetry invokes az.PublicIPAddressesClient.Delete with exponential backoff retry
-func (az *Cloud) DeletePublicIPWithRetry(pipName string) error {
+func (az *Cloud) DeletePublicIPWithRetry(pipResourceGroup string, pipName string) error {
```

Comment:
There's a minor style inconsistency - in CreateOrUpdate this is called `resourceGroup`, here `pipResourceGroup`.  (I think I slightly prefer the latter but don't feel strongly as long as they're consistent.)